deployment/influx_backup.md

```diff
@@ -0,0 +1,24 @@
+# Backup influxdb
+In this document we describe how to perform backup and restore of influxdb v2 from docker container.
+
+1. Backup the data from a docker container:
+`docker exec -it <container_id> influx backup <backup_location> -t <secret_token>`
```

Comment:
the secret token can be retrived from the `.env`?

Reply:
If needed yes but in this case the document describes how to perform the backup and restore manually from the server terminal. 
I think the idea is to setup cronjob with daily backup and we will use the restore only if needed. And maybe it is good idea to transfer the backup files on other location so we don't lose data if something happens with the whole server.

Reply:
thats okay, my question was only about the token, because i need to know where or how i have to retrive it.

Reply:
you can get it from the .env - yes